enterprise/server/test/integration/remote_execution/BUILD

```diff
@@ -9,7 +9,9 @@ go_test(
     # make these tests run faster
     timeout = "long",
     srcs = ["remote_execution_test.go"],
-    args = ["--test.v"],
+    exec_properties = {
+        "EstimatedComputeUnits": "5",
```

Comment:
Could you add `cpu:5` to tags for local scheduling?

Reply:
done